core/includes/install.core.inc

```diff
@@ -1808,14 +1815,14 @@ function _install_configure_form($form, &$form_state, &$install_state) {
   $form['site_information']['site_mail'] = array(
     '#type' => 'email',
     '#title' => st('Site e-mail address'),
-    '#default_value' => ini_get('sendmail_from'),
-    '#description' => st("Automated e-mails, such as registration information, will be sent from this address. Use an address ending in your site's domain to help prevent these e-mails from being flagged as spam."),
+    '#default_value' => $site_email,
+    '#description' => st("Automated outgoing e-mails, such as registration information, will be sent from this address. Use an email address ending in your site's domain, since using an email address from an external domain (such as a gmail.com or hotmail.com email) will result in e-mails being flagged as spam."),
     '#required' => TRUE,
     '#weight' => -15,
   );
   $form['admin_account'] = array(
     '#type' => 'fieldset',
-    '#title' => st('Site maintenance account'),
+    '#title' => st('Site administrator'),
```

Comment:
The word 'administrator' has a specific meaning in Backdrop. It's a role that can be assigned to anyone. Using that word here will be very confusing to people. How about "Master account"?